reconcile/external_resources/integration.py

```diff
@@ -23,14 +25,15 @@
     get_namespaces,
     get_settings,
 )
+from reconcile.utils.aws_api import AWSApi
```

Comment:
You may want to use `reconcile.utils.aws_api_typed` and implement `dynamodb`, wdyt?

Reply:
I implemented the basic structure for DynamodDB in the `aws_api_typed` and now the code uses the AWSStaticCredentials classes, etc. 

However, I don't see much value in re-implementing the CRUD methods in this layer as it's like duplicating code and types. I set a property to return the raw DynamoDB client instead. 

I left two commented methods in the DynamoDB implementation class for you to understand what I mean. We can discuss this approach if you want but I think, at least in this case, it's not worth adding put_item, del_tiem, get_item, etc. in the aws_api_typed. 

PTAL

Reply:
Yeah, it looks good, but please remove the commented/dead code from `reconcile/utils/aws_api_typed/dynamodb.py` 🙇

Reply:
removed.